exception at `list-selectors`

command:
```
node zenbot.js list-selectors
```

exception:
```
Error: Cannot find module '../extensions/exchanges/sim/products.json'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at list-selectors.js:13:24
    at Array.forEach (<anonymous>)
    at Command.<anonymous> (list-selectors.js:11:17)
    at Command.listener (node_modules\commander\index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)

--- a/commands/list-selectors.js
+++ b/commands/list-selectors.js
@@ -9,6 +9,8 @@ module.exports = function (program) {
     .action(function (/*cmd*/) {
       var exchanges = fs.readdirSync('./extensions/exchanges')
       exchanges.forEach(function(exchange){
+        if (exchange === 'sim' || exchange === '_stub') return
+
         console.log(`${exchange}:`)
         var products = require(`../extensions/exchanges/${exchange}/products.json`)
         products.sort(function (a, b) {

